core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -62,6 +62,11 @@ export interface IJestConfig {
    * Same as Jest CLI option testMatch
    */
   testMatch?: string[];
+
+  /**
+   * Indicate whether writing NUnit results is enabled when using the default reporter
+   */
+  writeNUnitResults?: boolean;
```

Comment:
maybe make more generic? "outputType: string" to open the possibilities for more results? also, thoughts on adding outputFilename?

Reply:
I think It'd be better to just leave this specific. We won't be building much more here in GCB.